compute/cker/include/cker/eigen/eigen_spatial_convolutions-inl.h

```diff
@@ -1245,9 +1274,6 @@ struct gemm_pack_rhs<
     eigen_assert(stride == 0);
     eigen_assert(offset == 0);
 
-    (void)stride;
-    (void)offset;
-
```

Comment:
@hseok-oh Without these lines, android release build got errors due to unused variables.